### DISCUSSION TITLE
Using Homespan in conjunction with another WiFi manager

### DISCUSSION THREAD

**Discussion by DonRonito** [HomeSpan/HomeSpan#493] [Q&A] [answered]
Can HomeSpan be configurered not to enable WiFi AP and not deal with the WiFi connection? I'm currently using another WiFi manager in my project, and would like to add HomeSpan - but the device is already connecting to WiFi without HomeSpan.

**Comment by DonRonito** [comment, marked as answer]
I believe I found a solution (?). Will check this out "void setApFunction(void (*func)())".

**Reply by HomeSpan** [reply]
Yes, this is correct.  The `setApFunction()` method was designed to allow you to replace HomeSpan's WiFi provisioning with your own logic to enter your WiFi credentials and connect to an Access Point.

**Reply by dgalue** [reply]
Hi, do you have any example where this function is used?
